src/Sulu/Component/Rest/ListBuilder/Metadata/SinglePropertyMetadata.php

```diff
@@ -16,10 +16,7 @@
  */
 class SinglePropertyMetadata extends AbstractPropertyMetadata
 {
-    /**
-     * @var ?FieldMetadata
-     */
-    private $field;
+    private ?FieldMetadata $field = null;
```

Comment:
This can be fixed. Because the setter and getter both are typed and the property is private. So unless you are doing something very unorthodox this shouldn't be a bc break.